vllm/v1/worker/gpu_model_runner.py

```diff
@@ -405,6 +406,7 @@ def load_model(self) -> None:
         if self.use_cuda_graph:
             # FIXME(woosuk): Currently, we do not use inductor to reduce the
             # compilation time and any potential issues with the inductor.
+            os.environ["VLLM_CUSTOM_OPS"] = "all"
```

Comment:
@ProExpertProg  we should improve the default option, basing on the value of `use_inductor` .